config/environments/development.rb

```diff
@@ -1,6 +1,7 @@
 # typed: false
 # frozen_string_literal: true
 
+# rubocop:disable Metrics/BlockLength
```

Comment:
I think we should replace these with an exception to Metrics/BlockLength for config/environments/*.rb in .rubocop.yml.

Reply:
No longer necessary as SMTP variables are no longer in config.